packages/components/accordion/src/accordion-item.tsx

```diff
@@ -51,48 +91,53 @@ export interface AccordionItemProps
     AccordionItemOptions {}
 
 export const AccordionItem = forwardRef<AccordionItemProps, "div">(
-  ({ className, children, icon, isDisabled = false, label, ...rest }, ref) => {
+  (
+    { className, children, disabled, icon, isDisabled = false, label, ...rest },
+    ref,
+  ) => {
     const itemId = useId()
     const panelId = useId()
 
-    const { index, isMultiple, isToggle, setFocusedIndex, setIndex, styles } =
+    const { index, multiple, setFocusedIndex, setIndex, styles, toggle } =
       useAccordionContext()
 
     const {
       descendants,
       index: i,
       register,
-    } = useAccordionDescendant({ disabled: isDisabled })
+    } = useAccordionDescendant({ disabled })
+
+    disabled ??= isDisabled
```

Comment:
I think it needs to be defined on line 103.

Reply:
Fixed